app/src/main/res/xml/xdrip_plus_prefs.xml

```diff
@@ -1198,7 +1181,6 @@
                 android:maxLength="4"
                 android:title="@string/title_pendiq_pin" />
 
-
         </PreferenceScreen>
         </PreferenceScreen>
```

Comment:
Here is still an indentation missing. Did you use code formatting?

Reply:
I didn't, no. I just manually corrected some of the more obvious ones. There's enough inconsistency in the formatting that the whitespace changes would have gotten out of hand really fast. As long as it was readable and more or less consistent with its surroundings, I tended not to mess with it.